src/v2/Apps/Consign/Components/SellArtDifferently.tsx

```diff
@@ -43,13 +52,22 @@ const Section: React.FC<{
   description: string
 }> = ({ icon, text, description }) => {
   return (
-    <Box width={["100%", "25%"]} height={["100%", 170]} my={[3, 0]} mx={2}>
+    <Box
+      width={["100%", "25%"]}
+      height={["100%", 170]}
+      my={[1, 0]}
+      textAlign="center"
+    >
       <Box>{icon}</Box>
-      <Box mt={1} mb={2}>
-        <Text variant="mediumText">{text}</Text>
+      <Box mt={[0, 1]} mb={[0, 2]}>
+        <Text variant="subtitle" color={color("black100")}>
```

Comment:
Our design system should automatically map to colors in theme, so 
```
color="black100"
```
(But another thought is that text is black100 by default)